feat/header component

ICarlosMarinho opened the pull request:
Refatora estilo do componente Button, implementa componentes Logo e Header.

ICarlosMarinho committed:
refactor: muda estilo do botão

ICarlosMarinho committed:
feat: implementa componente Logo

Implementa componente contendo o logo do site, o qual também serve como link
para a home page, além disso, implementa os estilos e testes do componente.

ICarlosMarinho committed:
feat: implementa componente Header

Implementa componente, bem como os estilos e os testes do componente.

ICarlosMarinho commented on src/components/Header/index.tsx:
```diff
@@ -0,0 +1,19 @@
+const Header = () => {
+  const navigate = useNavigate();
+
+  const handleButtonClick = () => navigate("/cart");
```
É necessário testar essa função?

michellehorn replied:
acho bacana. testa se, apos o click, a rota do navegador mudou

michellehorn commented on src/components/Header/test.tsx:
```diff
@@ -0,0 +1,33 @@
+import { MemoryRouter } from "react-router-dom";
+import Header from ".";
+
+describe("Header component | Unit test", () => {
```
🥳 nice

ICarlosMarinho committed:
test: adiciona testes para rotas

Implementa testes que verificam se a rota mudou corretamente ao clicar no
botão do carrinho e no logo.
